views/js/addons-connector.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License version 3.0
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
+ */
+
+const {$} = window;
+
+var mbo = {};
+
+(function() {
+    var pageMap = {
+        addonsLoginButtonSelector: '#addons_login_btn',
+        addonsConnectModalBtnSelector: '#page-header-desc-configuration-addons_connect',
+        addonsLogoutModalBtnSelector: '#page-header-desc-configuration-addons_logout',
+        addonsImportModalBtnSelector: '#page-header-desc-configuration-add_module',
+        addonsConnectModalSelector: '#module-modal-addons-connect',
+        addonsLogoutModalSelector: '#module-modal-addons-logout',
+        addonsConnectForm: '#addons-connect-form',
+    };
+
+    var AddonsConnector = function(pageMap) {
+        this.initConnect = function() {
+            // Make addons connect modal ready to be clicked
+            if ($(pageMap.addonsConnectModalBtnSelector).attr('href') === '#') {
+                $(pageMap.addonsConnectModalBtnSelector).attr('data-toggle', 'modal');
```

Comment:
```suggestion
                $(pageMap.addonsConnectModalBtnSelector).data('toggle', 'modal');
```
use dedicated method instead using attribute which are not really using the data collection